examples/static/css/style.css

```diff
@@ -1551,6 +1555,267 @@ button.xclose {
+}
+
+#posenet .close-button {
+  display: none;
+}
+
+#posenet #my-gui-container ul {
+  margin: 0px;
+  padding: 0px;
+  text-transform: capitalize;
+}
+
+#posenet #my-gui-container ul li {
+  list-style: none;
+  display: inline-block;
+  text-align: left;
+  margin: 0 0.5rem 0.1rem 0;
+}
+
+#posenet #my-gui-container ul li.folder {
+  margin-right: 0;
+}
+
+#posenet #my-gui-container ul li span {
+  margin-right: 0.5rem;
+}
+
+#posenet #my-gui-container ul li .title {
+  display: none;
+}
+
+#posenet #my-gui-container ul li input {
+  width: 40px;
+  border: 1px solid rgba(255, 111, 97, 0.1);
+  padding: 2px 6px;
+  background: transparent;
+}
+
+#posenet #my-gui-container ul li input:hover {
+  border: 1px solid rgba(255, 111, 97, 0.2);
+  background: rgba(255, 111, 97, 0.2);
+}
+
+#posenet #my-gui-container.fullscreen #gui {
+  padding: 1rem 1rem;
+  background-color: rgba(0, 0, 0, 0.1);
+}
+
+#posenet #my-gui-container.fullscreen .title {
+  margin-bottom: 0.2rem;
+}
+
+#posenet #my-gui-container.fullscreen ul li  {
+  display: block;
+}
+
+#posenet #my-gui-container.fullscreen ul li .title {
+  display: none;
+}
+
+#posenet #my-gui-container.fullscreen ul li input {
+  border: 1px solid rgba(255, 255, 255, 0.1);
+  color: rgba(255, 255, 255, 0.6);
+}
+
+#posenet #my-gui-container.fullscreen ul li input:hover {
+  border: 1px solid rgba(255, 255, 255, 0.1);
+  background: rgba(255, 255, 255, 0.2);
+}
+
+@media (max-width: 768px) {
+  #posenet #my-gui-container.fullscreen {
+    width: 60%;
+    margin: 1rem 1rem;
+    right: 1rem;
+  }
+
+  #posenet #canvasvideo {
+    width: 100%;
+    height: 100%;
+  }
+  
+  #posenet #fullscreen i svg {
+    left: 85%;
+  }
+}
+
+#posenet #showcanvas {
+  margin: 0 auto;
+}
+
+#posenet #my-gui-container.fullscreen ul li .c input[type=checkbox]:before {
+  border-color: rgba(255, 255, 255, 0.2);
+}
+
+#posenet #my-gui-container.fullscreen ul li .c input[type=checkbox]:hover::before {
+  border-color: rgba(255, 255, 255, 0.2);
+  background: rgba(255, 255, 255, 0.2);
+}
+
+#posenet #my-gui-container.fullscreen ul li select {
+  border: 1px solid rgba(255, 255, 255, 0.1);
+  color: rgba(255, 255, 255, 0.6);
+}
+
+#posenet #my-gui-container.fullscreen ul li select:hover,
+#posenet #my-gui-container.fullscreen ul li select:focus {
+  border: 1px solid rgba(255, 255, 255, 0.2);
+  background: rgba(255, 255, 255, 0.1);
+}
+
+#posenet #my-gui-container.fullscreen .select__arrow {
+  border-color: rgba(255, 255, 255, 0.1) transparent transparent transparent;
+}
+#posenet #my-gui-container.fullscreen ul li select:hover ~ .select__arrow,
+#posenet #my-gui-container.fullscreen ul li select:focus ~ .select__arrow {
+  border-top-color: rgba(255, 255, 255, 0.2);
+}
+#posenet #my-gui-container.fullscreen ul li select:disabled ~ .select__arrow {
+  border-top-color: rgba(255, 255, 255, 0.1);
+}
+
+#posenet ul li .c {
+  display: inline;
+}
+
+#posenet ul li .c input[type=checkbox] {
+  width: 0px !important;
```

Comment:
Setting checkbox's width to zero doesn't hide it on the Mac (Chrome 71)
<img width="323" alt="screen shot 2019-01-26 at 10 40 37 am" src="https://user-images.githubusercontent.com/7313938/51781665-d79ff800-2156-11e9-93af-d25a9886caea.png">

Reply:
Fixed in https://github.com/intel/webml-polyfill/pull/525/commits/20dceaa08524973f958fa60704ba07a44e4028ac